### DISCUSSION TITLE
CPU as backup, when Coral is not detected

### DISCUSSION THREAD

**Discussion by Wam-79** [blakeblackshear/frigate#7522] [Ideas]
Hi
I am running Frigate over Proxmox (qemu) and it works 70% times. Sometime after power outage, Frigate can't detect Coral (USB) and Docker container shutdown. I know that there is "way too many" conversations how Frigate-Proxmox-Coral(USB) combo not working. 

Is there any possible to get CPU as backup detector, if Coral not found? I want at least something triggered and logged and not realise after days that my Frigate is not running at all.

**Comment by NickM-27** [comment]
There is a pinned request for this https://github.com/blakeblackshear/frigate/issues/2904
